core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -103,6 +103,9 @@ def __init__(self,
         # temperature scaling parameter that is set by predictor if calibrate is true under TabularPredictor fit()
         self.temperature_scalar = None
 
+        # conformalize that is used to calibrate quantile estimates
```

Comment:
"conformalize that is used to calibrate quantile estimates" 
->
"whether to calibrate predictions via conformal methods"